### ISSUE TITLE
UI is unusable with too many / too long groups

### ISSUE THREAD

**Issue by thewoolleyman** [concourse/atc#39]
If you have too many groups and they wrap, they overlay the menu at a higher z-index, and items on the menu are unclickable.

I've created a bookmarklet with some styling hacks that helps some.  Before and after screenshots are attached.

Styling probably needs a holistic look, but perhaps at least the z-index/menu positioning fixes can be incorporated sooner than later to fix menu usability issues.

```
javascript:(function() {
  $('.nav-item, nav .groups li a').css('font-size', '12px', 'important');
  $('.nav-item, nav .groups li a').css('font-family', 'sans-serif', 'important');
  $('.nav-item, nav .groups li a').css('max-height', '40px', 'important');
  $('.nav-item, nav .groups li a').css('max-width', '150px', 'important');
  $('.nav-item, nav .groups li a').css('line-height', '40px', 'important');
  $('.nav-item, nav .groups li a').css('overflow', 'hidden', 'important');
  $('.nav-item, nav .groups li a').css('white-space', 'nowrap', 'important');
  $('.nav-item, nav .groups li a').css('text-overflow', 'ellipsis', 'important');
  $('.nav-item, nav .groups li a').css('padding', '0 2px', 'important');
  $('div .nav-container').css('z-index', '4', 'important');
  $('div .nav-container').css('top', '40px', 'important');
  $('.pipelines li').css('margin-top', '-34px', 'important');
})();
```

![screen shot 2015-12-29 at 12 14 33 am](https://cloud.githubusercontent.com/assets/158/12030835/f3b69370-adc1-11e5-941f-3952b7d8148a.png)
![screen shot 2015-12-29 at 12 14 13 am](https://cloud.githubusercontent.com/assets/158/12030834/f3b6269c-adc1-11e5-9dd6-02784491e1fe.png)

**Comment by thewoolleyman** [comment]
Discussed with Concourse team, and they said the preferred fix would be to make the groups list scroll horizontally, like the build numbers at the top of the build page.

**Comment by zachgersh** [comment]
Wanted to say, we are also experiencing the same sort of problem. See image below:
cc @kimeberz 

![screen shot 2016-05-23 at 9 57 03 am](https://cloud.githubusercontent.com/assets/689411/15478106/376d2b46-20cd-11e6-947c-e0473108137b.png)

**Comment by vito** [comment]
this was fixed by a PR we merged; should be in 2.8
